Exception during molecules loading in test fixture

**Summary**
When running tests for bingo got this exception during the fixture creation
Error: fingerprint/test_fingerprint.py [ERROR] Failed to read entity _Map_base::at, prev_molecule=MOL_1637

**Steps to Reproduce**
Run tests and see logs

**Additional context**
Happens in CI/CD

--- a/bingo/bingo-elastic/python/tests/conftest.py
+++ b/bingo/bingo-elastic/python/tests/conftest.py
@@ -1,3 +1,4 @@
+import gc
 import time
 from pathlib import Path
 from typing import Callable
@@ -30,7 +31,10 @@ def resource_loader() -> Callable[[str], str]:
 
 @pytest.fixture
 def indigo_fixture() -> Indigo:
-    return Indigo()
+    indigo = Indigo()
+    yield indigo
+    del indigo
+    gc.collect()
 
 
 @pytest.fixture

--- a/bingo/tests/conftest.py
+++ b/bingo/tests/conftest.py
@@ -1,5 +1,8 @@
+import gc
+
 import pytest
 from indigo import Indigo
+
 from .constants import (
     DATA_TYPES,
     DB_BINGO,
@@ -17,9 +20,10 @@ from .logger import logger
 
 @pytest.fixture(scope="class")
 def indigo():
-    # TODO: uncomment this:
     indigo = Indigo()
-    return indigo
+    yield indigo
+    del indigo
+    gc.collect()
 
 
 @pytest.fixture(scope="class")
@@ -48,6 +52,7 @@ def db(request, indigo):
         db.import_data(meta["import_no_sql"], data_type)
     elif db_str == DB_BINGO_ELASTIC:
         from bingo_elastic.elastic import IndexName
+
         from .dbc.BingoElastic import BingoElastic
 
         if data_type == EntitiesType.MOLECULES:

